packaging/AppImage/create_appimage.py

```diff
@@ -30,15 +30,23 @@ def copy_metadata_files(dist_path, version):
     Copy metadata files for the metadata of the AppImage.
     """
     copied_files = {
+        os.path.join("..", "icons", "cura-icon.svg"):         "usr/share/icons/hicolor/scalable/apps/cura-icon.svg",
+        os.path.join("..", "icons", "cura-icon_64x64.png"):   "usr/share/icons/hicolor/64x64/apps/cura-icon.png",
+        os.path.join("..", "icons", "cura-icon_128x128.png"): "usr/share/icons/hicolor/128x128/apps/cura-icon.png",
+        os.path.join("..", "icons", "cura-icon_256x256.png"): "usr/share/icons/hicolor/256x256/apps/cura-icon.png",
```

Comment:
```suggestion
{
        os.path.join("..", "icons", "cura-icon.svg"):         os.path.join("usr", "share", "icons", "hicolor", "scalable", "apps", "cura-icon.svg"),
        os.path.join("..", "icons", "cura-icon_64x64.png"):   os.path.join("usr", "share", "icons", "hicolor", "64x64", "apps", "cura-icon.png"),
        os.path.join("..", "icons", "cura-icon_128x128.png"): os.path.join("usr", "share", "icons", "hicolor", "128x128", "apps", "cura-icon.png"),
        os.path.join("..", "icons", "cura-icon_256x256.png"): os.path.join("usr", "share", "icons", "hicolor", "256x256", "apps", "cura-icon.png"),
```

It is always good practice to write path handling in a system agnostic way